### PR TITLE
Auth Decision tree

### PR DESCRIPTION
**Pull request by psiinon**
Sorry - its a big PR!

Initially based on https://github.com/psiinon/zap-auth-dt/wiki but it goes into more detail now.

No screenshots - too many to do...

### CODE REVIEW

File: site/content/docs/authentication/multiple-headers.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+You can do this via:
+* [Replacer Rules](/docs/authentication/handling-auth-yourself/#scripts) TODO fix
+* [Scripts](/docs/authentication/handling-auth-yourself/#replacer-rules) TODO fix
+
+
+##### Background
+
+You do have more than one authentication tokens which you need to supply in headers.
```
Comment (kingthorin) [inline site/content/docs/authentication/multiple-headers.md:14]: Either:
> Do you have more than one authentication token which you need to supply in the headers?

or

> Do you have multiple authentication tokens which you need to supply in the headers?

Comment (kingthorin) [reply site/content/docs/authentication/multiple-headers.md:14]: Or if it is really a statement:

> You have more than one authentication token which you need to supply in the headers.

File: site/content/docs/authentication/one-header.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+action: Configure ZAP to use your auth token
+---
+
+The easiest way to do this will be via [Authentication env vars](/docs/getting-further/authentication/handling-auth-yourself/#authentication-env-vars).
+
+Other options are available, as detailed on [Handling Authentication Yourself](/docs/getting-further/authentication/handling-auth-yourself/).
+
+##### Background
+
+You do have a single authentication token which you need to supply in a header.
```
Comment (kingthorin) [inline site/content/docs/authentication/one-header.md:14]: Assuming this is a statement:

> You have a single auth.....

File: site/content/docs/authentication/verification-not-identified.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+You may need to keep [Increasing the "Time to Wait"](../app-does-not-load) in order to give you enough time
+to explore the app.
+
+#### Next Step: [Manually Identify Verification Handling](../manual-verification/)
+
+If ZAP can still not identify the verification handling then you will need to do it manually.
+
+##### Background
+
+The Authentication Tester replies on the app to make enough background calls for it to identify the session handling and verification URL.
```
Comment (kingthorin) [inline site/content/docs/authentication/verification-not-identified.md:21]: relies

File: site/content/docs/authentication/multiple-headers.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+action: Configure ZAP to use your auth tokens
+---
+
+You can do this via:
+* [Replacer Rules](/docs/getting-further/authentication/handling-auth-yourself/#replacer-rules)
+* [Scripts](/docs/getting-further/authentication/handling-auth-yourself/#scripts)
+
+##### Background
+
+You have more than one authentication tokens which you need to supply in HTTP headers.
```
Comment (kingthorin) [inline site/content/docs/authentication/multiple-headers.md:14]: make tokens singular

Comment (psiinon) [reply site/content/docs/authentication/multiple-headers.md]: Had to read that twice to realise it was wrong ;) Done..

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**